oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyState.java

```diff
@@ -64,7 +65,9 @@ final class DocumentPropertyState implements PropertyState {
     private final byte[] compressedValue;
     private final Compression compression;
 
-    private static final int DEFAULT_COMPRESSION_THRESHOLD = Integer.getInteger("oak.mongo.compressionThreshold", -1);
+    private static final int DEFAULT_COMPRESSION_THRESHOLD = SystemPropertySupplier
+            .create("org.apache.jackrabbit.oak.plugins.document.DocumentPropertyState.stringCompressionThreshold", -1)
```

Comment:
It seems we were able to come up with different naming conventions for system properties in oak-store-document even:
* MemoryNodeStore uses `memoryds.`
* rdb uses `full.qualified.name`
* org.apache.jackrabbit.oak.plugins.document in general uses `oak.documentMk.`

Perhaps as this is in the main document package, should we use `oak.documentMk.stringCompressionThreshold` ?

Reply:
Sounds good to me.